src/dex-ui-components/base/Labels/MetricLabel.tsx

```diff
@@ -1,21 +1,66 @@
 import { Flex, Text, Skeleton } from "@chakra-ui/react";
+import { Color } from "../../themes";
 
 interface MetricLabelProps {
   label: string;
   value: string;
   isLoading?: boolean;
+  labelLeftIcon?: React.ReactElement;
+  labelRightIcon?: React.ReactElement;
+  labelTextColor?: string;
+  labelTextStyle?: string;
+  labelOpacity?: string;
+  valueTextColor?: string;
+  valueStyle?: string;
+  valueUnitSymbol?: string;
+  valueUnitSymbolColor?: string;
+  amount?: string;
+  amountLabelColor?: string;
 }
 
 export const MetricLabel = (props: MetricLabelProps) => {
   const { label, value, isLoading = false } = props;
 
   return (
     <Flex flexDirection="column" gap="1">
-      <Text textStyle="h4" opacity="0.8">
-        {label}
-      </Text>
+      <Flex flexDirection="row" alignItems="center" gap="1">
+        {props.labelLeftIcon ? props.labelLeftIcon : null}
+        <Text
+          textStyle={props.labelTextStyle ? props.labelTextStyle : "h4"}
```

Comment:
You can use prop destructing with default values to reduce the need for ternary operators:

`const { label, value, isLoading = false, labelTextStyle = "h4", labelTextColor = Color.Neutral._900, etc... } = props;`

It's a personal preference but I think the destructing approach is cleaner.

Reply:
Thanks for the helpful comment, I have addressed this.